modules/openapi-generator/src/main/resources/bash/README.mustache

```diff
@@ -38,6 +38,9 @@ $ echo '<body_content>' | {{scriptName}} --host <hostname> --content-type json <
 # }
 $ echo '<body_content>' | {{scriptName}} --host <hostname> --content-type json <operationId> key1==value1 key2=value2 key3:=23 -
 
+# Make POST request with form data
+$ {{scriptName}} --host <hostname> <operationId> key1:=value1 key2:=value2 key3:=23
+
```

Comment:
FYI. We can accept this enhancement to start with but 2 things:
- some APIs do not have POST methods
- users may have no clue what `operationId` is (I saw other part of the PR is already using it but I hope you see my point)


I think a better way is to support auto-generate documentation in the Bash client. Please let me know if you've time for the contribution and I can share some good starting points with you.

Reply:
Sure. I can work on the following enhancement.